KuringApp/KuringApp/Setting/AppIconSelectView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  AppIconSelectView.swift
+//  KuringApp
+//
+//  Created by 박성수 on 2023/09/27.
+//
+
+import SwiftUI
+import ComposableArchitecture
+
+enum KuringIcon: String, CaseIterable {
+    case kuring_app
+    case kuring_app_classic
+    case kuring_app_blueprint
+    case kuring_app_sketch
+    
+    var korValue: String {
+        switch self {
+        case .kuring_app: return "쿠링 기본"
+        case .kuring_app_classic: return "쿠링 클래식"
+        case .kuring_app_blueprint: return "쿠링 블루프린트"
+        case .kuring_app_sketch: return "쿠링 스케치"
+        }
+    }
+}
+
+struct AppIconSelect: Reducer {
+    struct State: Equatable {
+        var appIcons: [KuringIcon] = KuringIcon.allCases
+        var selectedIcon: KuringIcon?
+    }
+    
+    enum Action: Equatable {
+        case appIconButtonTapped(tappedValue: KuringIcon)
```

Comment:
```suggestion
        case appIconSelected(KuringIcon)
```
또는
```swift
appIconSelected(KuringIcon.ID)
```
두번째 코드는 "KuringIcon 전체를 넘길필요 없이 아이디만 넘겨줘도 되지 않을까?" 라는 생각에서 제안한 코드인데 아이디로 kuringIcons 들 조회해야해서 퍼포먼스는 첫번째가 더 나을 수도..?